pkg/metrics/metrics.go

```diff
@@ -66,29 +80,33 @@ func HandleTracking(cmd *cobra.Command, commandPath string, app *application.Ava
 	if !userIsOptedIn(app) {
 		return
 	}
-	if !cmd.HasSubCommands() && CheckCommandIsNotCompletion(cmd) {
-		trackMetrics(app, commandPath, flags)
+	if !app.Cmd.HasSubCommands() && CheckCommandIsNotCompletion(app.Cmd.CommandPath()) {
+		trackMetrics(app, flags, err)
 	}
 }
 
-func CheckCommandIsNotCompletion(cmd *cobra.Command) bool {
-	result := strings.Fields(cmd.CommandPath())
+func CheckCommandIsNotCompletion(commandPath string) bool {
+	result := strings.Fields(commandPath)
 	if len(result) >= 2 && result[1] == "completion" {
 		return false
 	}
 	return true
 }
 
-func trackMetrics(app *application.Avalanche, commandPath string, flags map[string]string) {
+func trackMetrics(app *application.Avalanche, flags map[string]string, cmdErr error) {
 	if telemetryToken == "" {
 		telemetryToken = os.Getenv(constants.MetricsAPITokenEnvVarName)
 	}
+	if telemetryToken == "" && !utils.IsE2E() {
+		app.Log.Warn("no token is configured for sending metrics")
+	}
 	if telemetryToken == "" || utils.IsE2E() {
 		return
 	}
-	client, _ := posthog.NewWithConfig(telemetryToken, posthog.Config{Endpoint: telemetryInstance})
-
-	defer client.Close()
+	client, err := posthog.NewWithConfig(telemetryToken, posthog.Config{Endpoint: telemetryInstance})
+	if err != nil {
+		app.Log.Warn(fmt.Sprintf("failure creating metrics client: %s", err))
+	}
```

Comment:
same here, why not just print nothing to logs?

Reply:
this is only going into avalanche.log, not stdout. we are seeing that not all events seem to be sent, so we need to have some logs on this, so we can see if something is happening and maybe try again based on the error